server/datastore/mysql/migrations/tables/20240110134315_AddDisplayVersionToOSTable.go

```diff
@@ -0,0 +1,42 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20240110134315, Down_20240110134315)
+}
+
+func Up_20240110134315(tx *sql.Tx) error {
+	addColumnStmt := `
+		ALTER TABLE operating_systems
+		ADD COLUMN display_version VARCHAR(10) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '';
```

Comment:
Why do we need a separate `display_version`? Why not add `22H2` as a suffix to the name? (from what I remember of the Figma screens)

Reply:
a downside to the stacked diff approach.  I plan on using that value in matching an OS with the MSRC product list.  Without going through all the Windows versions, if we just append the display_version onto the OS name (what osquery returns) I'm not sure the strings will match.  Figure it would be safer to `strings.Contains(productList, display_version)`